be/src/formats/orc/orc_mapping.cpp

```diff
@@ -85,20 +91,30 @@ Status OrcMapping::set_include_column_id_by_type(const OrcMappingPtr& mapping, c
     return Status::OK();
 }
 
-std::unique_ptr<OrcMapping> OrcMappingFactory::build_mapping(const std::vector<SlotDescriptor*>& slot_descs,
-                                                             const orc::Type& root_orc_type, const bool case_sensitve) {
+StatusOr<std::unique_ptr<OrcMapping>> OrcMappingFactory::build_mapping(
+        const std::vector<SlotDescriptor*>& slot_descs, const orc::Type& root_orc_type, const bool case_sensitve,
+        const bool use_orc_column_names, const std::vector<std::string>* hive_column_names) {
     std::unique_ptr<OrcMapping> orc_mapping = std::make_unique<OrcMapping>();
-    Status res = _init_orc_mapping(orc_mapping, slot_descs, root_orc_type, case_sensitve);
+    Status res;
+    if (use_orc_column_names) {
```

Comment:
Could be simplified as following:

```
res = _init_orc_mapping(orc_mapping, slot_descs, root_orc_type, case_sensitve, use_orc_column_names, hive_column_names).
```

and check `use_orc_column_names` inside.